src/endpoints/createLocalNewsElementEndpoint.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+
+import EndpointBuilder from '../EndpointBuilder'
+import type { JsonLocalNewsType } from '../types'
+import LocalNewsModel from '../models/LocalNewsModel'
+import moment from 'moment-timezone'
+import Endpoint from '../Endpoint'
+
+export const LOCALNEWS_ELEMENT_ENDPOINT_NAME = 'newsElement'
+
+type ParamsType = { city: string, language: string, id: number }
+
+export default (
+  baseUrl: string
+): Endpoint<ParamsType, Array<LocalNewsModel>> =>
+  new EndpointBuilder(LOCALNEWS_ELEMENT_ENDPOINT_NAME)
+    .withParamsToUrlMapper(
+      (params: ParamsType): string =>
+        `${baseUrl}/${params.city}/${params.language}/wp-json/extensions/v3/fcm?id=${params.id}`
+    )
+    .withMapper((json: Array<JsonLocalNewsType>): Array<LocalNewsModel> =>
```

Comment:
Shouldn't this only return one news element instead of an array? It should probably throw an error if the count is not exactly one. Not really sure though.

Reply:
The api response is an array with one item not an object.
https://cms-test.integreat-app.de/testumgebung/en/wp-json/extensions/v3/fcm?id=44

Reply:
Yeah sure, but couldn't you just return the first element?

Reply:
@klinzo What do you think?
```suggestion
    .withMapper((localNews: Array<JsonLocalNewsType>): Array<LocalNewsModel> => {
      const { id, timestamp, title, message } = localNews[0]
      return new LocalNewsModel({
        id,
        timestamp: moment.tz(timestamp, 'GMT'),
        title,
        message})
    })
```

Reply:
I'd add the following:
```suggestion
    .if (localNews.length !== 1) {
        throw new MappingError('Expected count of local news to be one. Received ${count} intsted')
    }
```
or something similar. @maxammann what do you think?

Reply:
@klinzo I Added that